devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -170,6 +170,10 @@ def refresh() {
 		return zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, [destEndpoint: 0x01])+
 		        zigbee.readAttribute(0x0402, 0x0000, [destEndpoint: 0x01])+
 		        zigbee.readAttribute(0x0405, 0x0000, [destEndpoint: 0x02])
+	} else if (isFrientSensor()) {
+		return zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020)+
+		        zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000)+
```

Comment:
Please correct spacing (use tab instead of spaces) in this file.

Reply:
changed spaces in to tabs